src/storage/src/hummock/sstable/block.rs

```diff
@@ -530,18 +542,19 @@ impl BlockBuilder {
 
         let mut key: BytesMut = Default::default();
         full_key.encode_into_without_table_id(&mut key);
```

Comment:
to mimic the real usage, let's do `encode_into` and then slice it with TABLE_ID_PREFIX here.